gnovm/stdlibs/runtime/runtime.gno

```diff
@@ -0,0 +1,35 @@
+// Package runtime allows access to the GnoVM's runtime information.
+package runtime
+
+func NewReadOnlyBanker() chain.Banker {
```

Comment:
I don't understand the benefit of having a distinct `runtime` package that ultimately implements helpers that will return `chain` types. Wouldn't it be simpler and less confusing to have a single package that defines the types and implements the helpers?